tests/FakeItEasy.Specs/EventRaisingSpecs.cs

```diff
@@ -526,5 +526,31 @@ public void RaisingInternalEvent(ClassWithInternalEventVisibleToDynamicProxy fak
             "Then the handler is called"
                 .x(() => A.CallTo(handler).MustHaveHappened());
         }
+
+        [Scenario]
+        public void SubscribeWithNullHandler(IEvents fake, Exception? exception)
+        {
+            "Given a fake with an event"
+                .x(() => fake = A.Fake<IEvents>());
+
+            "When subscribing to the event with a null handler"
+                .x(() => exception = Record.Exception(() => fake.UnsubscribedEvent += null));
+
+            "Then it should not throw"
+                .x(() => exception.Should().BeNull());
+        }
+
+        [Scenario]
+        public void UnsubscribeWithNullHandler(IEvents fake, Exception? exception)
+        {
+            "Given a fake with an event"
+                .x(() => fake = A.Fake<IEvents>());
+
+            "When unsubscribing from the event with a null handler"
+                .x(() => exception = Record.Exception(() => fake.UnsubscribedEvent -= null));
+
+            "Then it should not throw"
+                .x(() => exception.Should().BeNull());
+        }

```

Comment:
I don't love adding these specs to `EventRaisingSpecs`, which is about _raising_ events, not subscribing or unsubscribing. But we don't have specs for event subscription/unsubscription, and I also don't love adding a new spec file just for that.

Arguably, `EventRaisingSpecs` covers not only event raising, but also subscription/unsubscription. So maybe we should just rename it to `EventSpecs` or something like that.

Reply:
I agree. Renaming to `EventSpecs` suits me. We can do here or later, as pleases you.